### PR TITLE
Add Dodo MCB-USDC Strategy

### PR DESCRIPTION
**Pull request by magic990619**

### CODE REVIEW

File: src/tests/strategies/dodo/strategy-dodo-mcb-lp.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const {toWei} = require("../../utils/testHelper");
+const {getWantFromWhale} = require("../../utils/setupHelper");
+const {doTestBehaviorBase} = require("../testBehaviorBase");
+
+describe("StrategyDodoMcbUsdcLp", () => {
+  const want_addr = "0x34851ea13bde818b1efe26d31377906b47c9bbe2";
+
+  const whale_addr = "0x1001009911e3fe1d5b45ff8efea7732c33a6c012";
```
[the remark this answers is not in the document]
Comment (magic990619) [reply src/tests/strategies/dodo/strategy-dodo-mcb-lp.test.js]: I found one `whale_addr`. 
0x1825b9faf7c3ab0efbbd927cd1a8fe2c86c933fd

File: src/strategies/arbitrum/dodo/strategy-dodo-mcb-usdc.sol [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+import "../strategy-base.sol";
+import "../../../interfaces/dodo.sol";
+
+contract StrategyDodoMcbUsdcLp is StrategyBase {
+    // Token addresses
+    address public constant dodo = 0x69Eb4FA4a2fbd498C257C57Ea8b7655a2559A581;
+    address public constant mcb = 0x4e352cF164E64ADCBad318C3a1e222E9EBa4Ce42;
+    address public constant usdc = 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8;
+
+    address public constant rewards =
```
[the remark this answers is not in the document]
Comment (magic990619) [reply src/strategies/arbitrum/dodo/strategy-dodo-mcb-usdc.sol:15]: Found this one.
0x98CEb851aF3d8627287885D56AEA863B848CeB6F
https://dodoex.github.io/docs/docs/mining/#arbitrum-one

File: src/strategies/arbitrum/dodo/strategy-dodo-mcb-usdc.sol [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                _dodo,
+                1,
+                path,
+                1,
+                false,
+                now + 60
+            );
+        }
+        
+        uint256 _mcb = IERC20(mcb).balanceOf((address(this)));
```
[the remark this answers is not in the document]
Comment (magic990619) [reply src/strategies/arbitrum/dodo/strategy-dodo-mcb-usdc.sol]: Fixed

**Comment by larrythecucumber321** [comment]
Rewards no longer available for this pair. Closing PR.

**Pull request closed without merging**